packages/arb-token-bridge-ui/src/state/app/utils.ts

```diff
@@ -71,6 +103,16 @@ export const getDepositStatus = (tx: Transaction | MergedTransaction) => {
   }
 }
 
+export const isRetryableTicketFailed = (
+  retryableTicketStatus: L1ToL2MessageStatus
+) => {
+  return (
+    retryableTicketStatus === L1ToL2MessageStatus.CREATION_FAILED ||
+    retryableTicketStatus === L1ToL2MessageStatus.EXPIRED ||
```

Comment:
I have removed the case of `L1ToL2MessageStatus.EXPIRED` because this will happen after 7 days of not redeeming the retryable manually, and then there's nothing we'd be able to do. But the other 2 cases - creation-failed or `FUNDS_DEPOSITED_ON_L2` are actionable items and they hold true for retryable-failure case.